containers/codespaces-linux/.devcontainer/Dockerfile

```diff
@@ -54,6 +54,12 @@ RUN apt-get update -yq\
     # Be sure git is up to date due to DSA 4659-1, DSA 4657-1
     && apt-get upgrade -yq git \
     #
+    # Install Git CLI v 0.11.0
+    && echo "Downloading github CLI" \
+    && curl -OL https://github.com/cli/cli/releases/download/v0.11.0/gh_0.11.0_linux_amd64.deb \
+    && echo "Installing github CLI" \
+    && apt install ./gh_0.11.0_linux_amd64.deb \
```

Comment:
We need to delete the .deb file from disk after the install

Reply:
updated code to remove the deb file after installation. Now the file is deleted after installation. I have verified in the container.